### ISSUE TITLE
Use of JMeter Include controller and use of top level variables in included file

### ISSUE THREAD

**Issue by ngordon-d2l** [awslabs/distributed-load-testing-on-aws#56]
**What is your question?**
I have a main jmeter script that references another script via the jmeter include controller.
This works when run outside the solution and variable values are passed but theres issue via the solution.

For the solution, the zip file contains the main jmx, plus a folder with the included jmx.
When run in the solution I can see that the other script is accessed using debug statements, but a variable set in the first is not set in the second.

Here's how I do it:
first jmx: in JSR223 Sampler/groovy: the value is set using vars.put("var", varvalue);
second jmx: accessed as ${var}
Can you tell me if this is supported?

Recent update from Diego Magalhaes - our AWS contact - indicates the following:
It goes through the files in zip file to find the first JMX file, so if JMX 1 is shown first, the solution will run JMX 1, and if JMX 2 is shown first, the solution will run JMX 2 instead which would cause an issue for the customer.

To summarize what I'm looking for based on the above:
- how to make sure my root jmx is seen first?
- how to pass the variable from the root to included jmx  (could it be solved by just ensuring the root is seen first??)

I'm prepping for an upcoming series of very high load tests - I would appreciate any insight you can provide on this.
Thx, Nicola

**Comment by ngordon-d2l** [comment]
Just an update on getting that test running in the solution 
I am able to get the root jmx selected, and it includes jmx files in folders - a very simple example currently.  
Here's the things that solved my issues:
the solution picks first jmx by name alphabetically - name includes path, so folder names matter.
the solution runs on linux, so included files must be referenced by linux path format.

No need to answer the submitted question - all good, thx  :-)
